src/styles/Blocks/Header.scss

```diff
@@ -0,0 +1,66 @@
+header {
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/ea3cf15f-7967-4980-89d3-668730ad44de)